wordpress-mobile/WordPress-iOS
# [Stats Refresh] Insights detail view async loading

Fixes #12424 

This PR removes the NRV from the Insights detail view and uses the async loading + ghost view approach instead.
The section affected are:
• _Followers_
• _Comments_
• _Tags & Categories_
• _This Year_

## To test:
• Select Stats from a site with data (en.blog is a good candidate) and select _Insights_
• Make sure the sections mentioned above are present in _Insights_
• For every section click on _View more_ and open the detail
• You should see the Ghost views before the content is loaded

Update release notes:
